modules/history/src/views/full/MessageViewer.jsx

```diff
@@ -165,55 +111,80 @@ export class MessageViewer extends React.Component {
     this.props.updateConversationWithFilters(f)
   }
 
-  render() {
-    if (!this.props.conversation) {
-      return <NoConversationSelected />
-    }
+  getLastMessageDate = messageGroups => {
+    const maxDateMessage = _.maxBy(messageGroups, mg => mg.userMessage.createdOn)
+    return new Date(maxDateMessage.userMessage.createdOn)
+  }
+
+  renderHeader() {
     return (
-      <div className={style['message-viewer']}>
-        <div
-          className={classnames(
-            style['message-list'],
-            this.state.inspectorIsShown ? style['message-list-partial'] : style['message-list-full']
-          )}
-        >
-          <MessagesHeader conversation={this.state.currentConversation} messageGroups={this.props.messageGroups} />
-          <MessagesTaskBar
-            ref="taskBar"
-            selectedCount={this.state.selectedGroups.length}
-            useAsFilter={!this.state.areMessagesSelected}
-            flag={this.flagSelectedMessages}
-            unflag={this.unflagSelectedMessages}
-            updateFilters={this.updateFilters}
-            currentConv={this.state.currentConversation}
-          />
+      this.state.currentConversation && (
+        <div>
```

Comment:
If the <div> only serves for returning multiple components and doesn't have any rendering usefulness, you can use <React.Fragment> instead